include/drivers/at_cmd.rst

```diff
@@ -0,0 +1,51 @@
+﻿.. _at_cmd_readme:
+
+AT command interface
+####################
+
+The socket interface has a global limitation of 8 socket instances that can be
+open at the same time.
+So if an application needs multiple socket protocols like TCP,
+UDP, TLS, AT, and so on, the available 8 socket instances become a limiting factor.
+The AT command interface driver is designed to let multiple clients, i.e.,
+threads, software modules/components to share a single AT socket instance.
```

Comment:
The important part here is actually just "threads". So maybe it is enough to say ".... multiple threads share ....", and drop the content between the parenthesis. Because that is what must be synchronized.

Reply:
Updated.